src/components.d.ts

```diff
@@ -151,11 +151,6 @@ export namespace Components {
 
 declare global {
 
-  // Adding a global JSX for backcompatibility with legacy dependencies
```

Comment:
this deletion keeps happening when i 'npm run start' locally. I remove it - should you?

Reply:
It shouldn't be there in the first place, I'm not sure how it got added back but it was removed by Stencil.